pages/docs/install/install-mac.md

```diff
@@ -17,7 +17,7 @@ brew cask install vagrant
 brew cask install vagrant-manager
 
 # Create a working directory for the Vagrant configuration and
-# generate a template Vagrantfile for "ubuntu/xenial64"
+# generate a template Vagrantfile for "bento/ubuntu-16.04"
 mkdir singularity-vm
 cd singularity-vm
 vagrant init ubuntu/xenial64
```

Comment:
hey @alaindomissy let me know if you want to fix this - I'd be happy to do a quick PR to change it on my end, since it's pretty small. Let me know!